service/history/queueTask.go

```diff
@@ -33,17 +33,18 @@ import (
 	"github.com/uber/cadence/common/metrics"
 	"github.com/uber/cadence/common/persistence"
 	"github.com/uber/cadence/common/service/dynamicconfig"
-	"github.com/uber/cadence/common/task"
+	ctask "github.com/uber/cadence/common/task"
```

Comment:
This is temporary. Ideally, only files in `history/task` package will import `common/task` package. I will move files that are currently in `history` package and imports `common/task` to `history/task`